server/routes/posts.js

```diff
@@ -120,6 +120,13 @@ router.put("/:postID", function(req, res) {
     });
 });
 
+// Delete post
+router.delete("/:postID", function(req, res) {
+  Post.findByIdAndDelete(req.params.postID)
+    .then(() => res.json("Post deleted."))
+    .catch(err => res.status(400).json("Error: " + err));
+});
```

Comment:
This seems fine.  More a question for Aecio; does casting the error response as a .json cost anything?  i.e. Is there any difference between res.status(400).json("Error: " + err) and res.status(400).send({error: err}) ?

Reply:
It's casted by a middleware already, so it will have the same result think.